### DISCUSSION TITLE
Debugging without adb?

### DISCUSSION THREAD

**Discussion by xpseudonym** [juha-h/baresip-studio#501] [Q&A] [answered]
The [debug page on the wiki](https://github.com/juha-h/baresip-studio/wiki/Debugging) describes the convoluted process of setting up `adb` etc in order to debug, which is all a bit inconvenient when I'm on the go using my device.
Is there anyway to run the debug options on the phone - output `>` to some accessible file somewhere to collect the random calls etc that consist of everyday use?
Thanks

**Comment by juha-h** [comment]
I have not tried, but found this link about adb in termux:

https://www.macrodroidforum.com/index.php?threads/guide-setting-up-and-using-adb-commands-with-termux.5640/#:~:text=%5BMarker%20for%20those%20who%20wish%20to%20automate%20everything%20on%20Android%2011%2B%5D

**Reply by xpseudonym** [reply]
Thanks @juha-h 
This is where I got to:
http://www.macrodroidforum.com/index.php?threads/guide-setting-up-and-using-adb-commands-with-termux.5640/post-50776

**Reply by xpseudonym** [reply, marked as answer]
https://github.com/juha-h/baresip-studio/issues/506#issuecomment-2446524924
